Core/src/org/sleuthkit/autopsy/discovery/ui/DiscoveryTopComponent.java

```diff
@@ -267,6 +276,25 @@ private void newSearchButtonActionPerformed(java.awt.event.ActionEvent evt) {//G
                 .collect(Collectors.toList());
     }
 
+//    /**
```

Comment:
Is this here for a later time or should it be deleted?

Reply:
Will delete, it was part of an earlier attempt to make the panels reset correctly when nothing was selected that I forgot to remove.